handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslContext.java

```diff
@@ -132,7 +142,8 @@ protected void deallocate() {
     final Certificate[] keyCertChain;
     final ClientAuth clientAuth;
     final OpenSslEngineMap engineMap = new DefaultOpenSslEngineMap();
-    volatile boolean rejectRemoteInitiatedRenegotiation;
+    private volatile boolean rejectRemoteInitiatedRenegotiation;
+    private volatile int maxBioSize = MAX_BIO_SIZE;
```

Comment:
here... expand the diff:
https://github.com/netty/netty/pull/6123/files#diff-cfd417915faad637e4278b54adafd9baR275